reverse_proxy/defaults/main.yml

```diff
@@ -5,3 +5,7 @@ zone: tre.example.org
 # cran mirror
 cran_mirror: https://cran.ma.imperial.ac.uk/
 cran_regex: ^/(bin/windows|src)/contrib/(4\.1|4\.3|3\.6)*/*[a-zA-Z0-9_\-\.]*(\.tar\.gz|\.zip|PACKAGES)$
+
+# conda-forge
+conda_repo_regex: ^/conda-forge/(linux-64|linux-aarch64|win-64|noarch)/(current_)*repodata.json$
+conda_pkg_regex: ^/conda-forge/(linux-64|linux-aarch64|win-64|noarch)/[a-zA-Z0-9\-\._!]*(\.tar\.bz2|\.conda)$
```

Comment:
What's the significance of `!` in the regex? Is it used in filenames or does it mean something else?

Reply:
Yes :(

e.g. x264-1!164.3095-h4e544f5_2.tar.bz2

Reply:
It's to denote an epoch https://docs.conda.io/projects/conda/en/latest/user-guide/concepts/pkg-specs.html#version-ordering